Source/ACE.Server/Factories/LootGenerationFactory_Jewelry.cs

```diff
@@ -98,79 +99,72 @@ private static WorldObject CreateJewels(int tier, bool isMagical)
         private static WorldObject CreateJewelry(TreasureDeath profile, bool isMagical)
         {
 
-            // 35% chance ring, 35% chance bracelet, 30% chance necklace
-            int ringPercent = 35;
-            int braceletPercent = 35;
+            // 31% chance ring, 31% chance bracelet, 30% chance necklace 8% chance Trinket
+            int ringPercent = 31;
+            int braceletPercent = 31;
             int necklacePercent = 30;
+            int trinketPercent = 8;
 
-            int jewelrySlot = ThreadSafeRandom.Next(0, ringPercent + braceletPercent + necklacePercent);
+            int jewelrySlot = ThreadSafeRandom.Next(0, ringPercent + braceletPercent + necklacePercent + trinketPercent);
```

Comment:
it looks like these #s are intended to add up to 100%

however, it is rolling with 101 range (0-100), instead of (1-100)

Reply:
yea, I am just going to get rid of all of that